docs/book/v4/strategy.md

```diff
@@ -82,6 +82,26 @@ if present, will use it to translate the property name prior to looking up a
 
 ## Available implementations
 
+### Laminas\\Hydrator\\Strategy\\BackedEnumStrategy
+
+**PHP 8.1+** This strategy coverts scalar values into [Backed Enums](https://www.php.net/manual/en/language.enumerations.backed.php)
+and visa versa:
+
+```php
+enum Direction: string
+{
+    case Left  = 'left';
+    case Right = 'right';
+}
+
+$enumStrategy = new Laminas\Hydrator\Strategy\BackedEnumStrategy(Direction::class);
+
+$case = $enumStrategy->hydrate('right', null);
+// enum Direction::Right : string("right");
+$direction = $enumStrategy->extract(Direction::Left);
+// string(4) "left"
+```
+
```

Comment:
We need a separate page for the new hydrator strategy which contains:

* with the notice "Available since version 4.8.0"
* with the notice "Installation Requirements PHP 8.1"
* an explanation of the strategy
* basic usage with code example that shows standalone usage without adding the strategy to a hydrator
  * Create and configure strategy
  * Hydrate data
  * Extract data
* Example with a hydrator and class
  * Create hydrator and add strategy
  * Hydrate data
  * Extract data


Compare with:

* https://docs.laminas.dev/laminas-hydrator/v4/strategies/collection/
* https://docs.laminas.dev/laminas-hydrator/v4/strategies/datetime-immutable-formatter-strategy/
* https://docs.laminas.dev/laminas-hydrator/v4/strategies/hydrator/
* https://docs.laminas.dev/laminas-hydrator/v4/strategies/serializable/

Add the new page to configuration file for docs:

https://github.com/laminas/laminas-hydrator/blob/fcaa066c003698323e04efbaf2ff1e21ef74ca8c/mkdocs.yml#L15-L20

Thanks in advance! 👍🏻

Reply:
> with the notice "Available since version 4.8.0"

Perhaps too meta for this particular patch, but perhaps we can find a better solution for this "since" stuff? :thinking: 

 > with the notice "Installation Requirements PHP 8.1"

This is already part of this documentation

Reply:
> Perhaps too meta for this particular patch, but perhaps we can find a better solution for this "since" stuff? 🤔

We have consistently included the same note across all documentation.

Example:

```markdown
INFO: **New Feature**
Available since version 4.8.0
```

```markdown
MISSING: **Installation Requirements**
[Enumerations](https://www.php.net/manual/language.enumerations.overview.php) require PHP version 8.1 or higher.
```

https://oprypin.github.io/markdown-callouts/

Reply:
@froschdesign I can't actually see callouts in the mkdocs.yml plugins section? 

I've followed the style of datetime-immutable-formatter-strategy.md, with ">" for the "Available since" and "Requires".

Reply:
> I can't actually see callouts in the mkdocs.yml plugins section?

Will be added during documentation build. See: https://github.com/laminas/documentation-theme/blob/c41e6389c1283f17179f7730b423b14b4aff676e/update_mkdocs_yml.py#L19-L36